imports/components/@enhancers/recently-liked/index.js

```diff
@@ -0,0 +1,27 @@
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
+
+const RECENT_LIKES_QUERY = gql`
+  {
+    recentlyLiked(limit: 10, skip: 0) {
+      id
+      ... on Content {
```

Comment:
@jbaxleyiii Sure thing, this was mainly a commit to end the day, and keep you in the loop.  @JakeDawkins and I had plans to sync tomorrow about the enhancer, to ensure that we were able to pass variables to it.  I'll take a look through the PR you commented on from Jake as well.